src/sidebar/services/test/api-test.js

```diff
@@ -17,9 +17,11 @@ const util = require('../../../shared/test/util');
 const routes = require('./api-index.json').links;
 
 describe('sidebar.services.api', function () {
-  let $httpBackend = null;
-  let sandbox = null;
-  let api = null;
+  let $httpBackend;
+  let $q;
+  let fakeAuth;
+  let sandbox;
+  let api;
```

Comment:
The `= null` part was left-over from a conversion from CoffeeScript, which has no way to declare variables without initializing them AFAIK.